lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -241,6 +241,19 @@ sub machine {
     return $self->settings_hash->{MACHINE};
 }
 
+sub obs_url() {
+    return "https://build.opensuse.org/";
+}
```

Comment:
sorry, but this is nonsense. The webui just can't know the source project.

a) we use 42:S:A for openSUSE:42 - and we might add random other names for other projects
b) we use Staging:A from SP1 stagings, but they are *not* on build.opensuse.org

This is just doomed to fail.